aoi-db/Dockerfile

```diff
@@ -0,0 +1,7 @@
+# Get MySQL image from Docker Hub
+
+FROM mysql
+
+ENV MYSQL_ROOT_PASSWORD="password"
```

Comment:
I believe this should be "example" not "password" to keep with the same convention we have had